src/operations-tracker-header/operations-tracker-header.tsx

```diff
@@ -34,7 +34,18 @@ export const OperationsTrackerHeader = React.memo(
       operationsState,
       onCopy,
     } = useOperationsTrackerheader(props);
+    const trackerStore = React.useContext(TrackerStoreContext);
+    const {
+      setTheme, theme
+    } = useStore(trackerStore, (store) => ({
+      setTheme: store.setTheme,
+      theme: store.theme,
+    }));
 
+    const changeTheme = (e: any)=>{
+      console.log(theme);
+      setTheme(e.target.value)
+    }
```

Comment:
Move this inside useOperationsTrackerHeader function and also extract it to its own hook